src/components/pages/home/index.tsx

```diff
@@ -47,143 +46,11 @@ const Home: React.FC<any> = ({data, jumbotron, location}) => {
-                              />
-                            )
-                          case 7:
-                            return i === 0 ? (
-                              <HorizontalResourceCard
-                                className="col-span-2"
-                                key={resource.id}
-                                resource={resource}
-                                location={location}
-                                completedCoursesIds={completedCoursesIds}
-                              />
-                            ) : (
-                              <VerticalResourceCard
-                                key={resource.id}
-                                resource={resource}
-                                location={location}
-                                completedCoursesIds={completedCoursesIds}
-                              />
-                            )
-                          default:
-                            return (
-                              <VerticalResourceCard
-                                key={resource.id}
-                                resource={resource}
-                                location={location}
-                                completedCoursesIds={completedCoursesIds}
-                              />
-                            )
-                        }
-                      },
-                    )}
-                  </Grid>
-                  {section.path && (
-                    <div className="flex justify-end mt-3">
-                      <Link href={section.path} passHref>
-                        <a
-                          onClick={() => {
-                            analytics.events.activityInternalLinkClick(
-                              'curated topic page',
-                              'home page-curated section',
-                              section.title,
-                              section.path,
-                            )
-                          }}
-                          className="flex items-center px-4 py-3 text-sm transition-all duration-200 ease-in-out bg-transparent border-b-2 border-gray-200 dark:border-gray-800 border-opacity-70 dark:hover:bg-gray-800 dark:hover:bg-opacity-50 opacity-80 hover:opacity-100 group"
-                        >
-                          Browse all{' '}
-                          <span
-                            className="pl-1 transition-all duration-200 ease-in-out group-hover:translate-x-1"
-                            aria-hidden
-                          >
-                            →
-                          </span>
-                        </a>
-                      </Link>
-                    </div>
-                  )}
-                </section>
-              )
-            })}
+          <SanitySections
+            sections={data.sections}
+            location={location}
+            completedCoursesIds={completedCoursesIds}
+          />
```

Comment:
most of the home page is just the `SanitySections` which is extracted to it's own component